packages/reason-relay/src/ReasonRelay.re

```diff
@@ -37,9 +34,9 @@ external isClientID: dataId => bool = "isClientID";
  */
 
 // We occasionally have to remove undefined keys from objects, something I haven't figured out how to do with pure BuckleScript
-let _cleanObjectFromUndefined = [%bs.raw
+let _cleanObjectFromUndefined = [%raw
   {|
-  function cleanObj(obj) {
+  function (obj) {
```

Comment:
Any particular reason for removing the function names? Are names added to the function anyway through raw?

Reply:
yes, I removed the function name because it prevents BS JS parsing system from correctly inferring the function arity, without it it doesn't use `Curry` in the generated output. The reason function name is added anyway.

Reply:
Excellent, did not know that!